clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/agent/ReservationReportCachingAgent.groovy

```diff
@@ -62,13 +69,19 @@ class ReservationReportCachingAgent implements CachingAgent {
   final AmazonClientProvider amazonClientProvider
   final Collection<NetflixAmazonCredentials> accounts
   final ObjectMapper objectMapper
+  final AccountReservationDetailSerializer accountReservationDetailSerializer
 
   ReservationReportCachingAgent(AmazonClientProvider amazonClientProvider,
                                 Collection<NetflixAmazonCredentials> accounts,
                                 ObjectMapper objectMapper) {
     this.amazonClientProvider = amazonClientProvider
     this.accounts = accounts
-    this.objectMapper = objectMapper.enable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+
+    def module = new SimpleModule()
+    accountReservationDetailSerializer = new AccountReservationDetailSerializer()
+    module.addSerializer(AmazonReservationReport.AccountReservationDetail.class, accountReservationDetailSerializer)
```

Comment:
Added a custom serializer that can either separate or merge vpc and non-vpc reserved instance usage.

```
"prod": {
    "reserved": 22,
    "used": 12,
    "surplus": 10
  }
```

vs

```
"prod": {
    "reserved": 12,
    "used": 2,
    "surplus": 10,
    "reservedVpc": 10,
    "usedVpc": 8,
    "surplusVpc": 2
  }
```